### PR TITLE
Add spotless check in CI

### PR DESCRIPTION
**Pull request by Ao-senXiong**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -36,3 +36,5 @@ jobs:
           echo "REPO_SITE=${owner}" >> $GITHUB_ENV
       - name: Build with Gradle
         run: ./gradlew build --warning-mode all
+      - name: Spotless Check
+        run: ./gradlew spotlessCheck
```
Comment (wmdietl) [inline .github/workflows/main.yml:40]: Doesn't this already happen with `build`? Or should we change `build` to `assemble`?

Comment (Ao-senXiong) [reply .github/workflows/main.yml]: You are right. The build consists of spotlessCheck. I have updated the PR with separate tasks. If you think use `./gradlew build` is clearer, we may close this PR.

**Review by wmdietl — COMMENTED** [review]
Isn't it usually quite easy to tell if an error comes from Spotless? We don't do this in the other projects and this seems like more lines with a risk of missing something that `build` would automatically do... So unless you have an argument why it is much clearer like this, I think we should close this.

**Comment by Ao-senXiong** [comment]
> Isn't it usually quite easy to tell if an error comes from Spotless? We don't do this in the other projects and this seems like more lines with a risk of missing something that `build` would automatically do... So unless you have an argument why it is much clearer like this, I think we should close this.

Ok, I will close this PR.

**Pull request closed without merging**